File: reactapp4.client/src/interfaces/BoxScoreScoring.ts
interface BoxScoreScoring {
    game_id: string;
    team_id: string;
    team_abbreviation: string;
    team_city: string;
    player_id: string;
    player_name: string;
    nickname: string;
    start_position: string;
    comment: string;
    min: number;
    pct_fga_2pt: number;
    pct_fga_3pt: number;
    pct_pts_2pt: number;
    pct_pts_2pt_mr: number;
    pct_pts_3pt: number;
    pct_pts_fb: number;
    pct_pts_ft: number;
    pct_pts_off_tov: number;
    pct_pts_paint: number;
    pct_ast_2pm: number;
    pct_uast_2pm: number;
    pct_ast_3pm: number;
    pct_uast_3pm: number;
    pct_ast_fgm: number;
}

export type { BoxScoreScoring };

File: ReactApp4.Server/Data/BoxScoreScoring.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace ReactApp4.Server.Data
{
    public class BoxScoreScoring
    {
        [Column("id")]
        public int Id { get; set; }

        [Column("game_id")]
        public string? Game_id { get; set; }

        [Column("team_id")]
        public string? Team_id { get; set; }

        [Column("team_abbreviation")]
        public string? Team_abbreviation { get; set; }

        [Column("team_city")]
        public string? Team_city { get; set; }

        [Column("player_id")]
        public string? Player_id { get; set; }

        [Column("player_name")]
        public string? Player_name { get; set; }

        [Column("nickname")]
        public string? Nickname { get; set; }

        [Column("start_position")]
        public string? Start_position { get; set; }

        [Column("comment")]
        public string? Comment { get; set; }

        [Column("min")]
        public decimal? Min { get; set; }

        [Column("pct_fga_2pt")]
        public decimal? Pct_fga_2pt { get; set; }

        [Column("pct_fga_3pt")]
        public decimal? Pct_fga_3pt { get; set; }

        [Column("pct_pts_2pt")]
        public decimal? Pct_pts_2pt { get; set; }

        [Column("pct_pts_2pt_mr")]
        public decimal? Pct_pts_2pt_mr { get; set; }

        [Column("pct_pts_3pt")]
        public decimal? Pct_pts_3pt { get; set; }

        [Column("pct_pts_fb")]
        public decimal? Pct_pts_fb { get; set; }

        [Column("pct_pts_ft")]
        public decimal? Pct_pts_ft { get; set; }

        [Column("pct_pts_off_tov")]
        public decimal? Pct_pts_off_tov { get; set; }

        [Column("pct_pts_paint")]
        public decimal? Pct_pts_paint { get; set; }

        [Column("pct_ast_2pm")]
        public decimal? Pct_ast_2pm { get; set; }

        [Column("pct_uast_2pm")]
        public decimal? Pct_uast_2pm { get; set; }

        [Column("pct_ast_3pm")]
        public decimal? Pct_ast_3pm { get; set; }

        [Column("pct_uast_3pm")]
        public decimal? Pct_uast_3pm { get; set; }

        [Column("pct_ast_fgm")]
        public decimal? Pct_ast_fgm { get; set; }
    }
}
